apps/src/p5lab/spritelab/blocks.js

```diff
@@ -40,6 +40,11 @@ function animations(areBackgrounds) {
         return [url, `"${animation.name}"`];
       }
     });
+  // In case either all backgrounds or all costumes are missing and we request them
+  if (results.length === 0) {
+    return [['sprites missing', 'null']];
+  }
```

Comment:
What's the user experience here?  is it worth noting in the comment?

Reply:
the block will show up but the dropdown of backgrounds or sprites will be empty and they'll just have to add sprites themselves using the "more" button. I can add it to the comment!